packages/storybook-react/stories/i18n/remirror-i18n.stories.tsx

```diff
@@ -0,0 +1,5 @@
+import Basic from './remirror-i18n';
+
+export default { title: 'i18n / @remirror⁄i18n' };
```

Comment:
The character between `@remirror` and `i18n` is weird.

Reply:
Yeah, unfortunately adding a normal slash character makes Storybook split it into a sub folder. It seems Storybook no longer allows you to change this.